fast_and_reliable.py

```diff
@@ -0,0 +1,37 @@
+# C:\Users\IMOE001\Downloads\words.txt
+import re
+from time import process_time
+import typing
+
+
+def fast_and_reliable(path: str) -> None:
+    """
+    A function that checks how long on average a word search in a list and group lasts.
```

Comment:
Notice that the translation of קבוצה in python is set, and not a group.
(They are two different mathematical objects - https://en.wikipedia.org/wiki/Group_(mathematics)   https://en.wikipedia.org/wiki/Set_(mathematics) ).